lib/west-commands/uv.py

```
@@ -181,7 +181,7 @@ def do_run_pip(self, args, manager_args):
 
             if len(requirements) > 0:
                 subprocess.check_call(
-                    ["uv", "pip", "install"]
+                    [os.environ["UV"], "pip", "install"]
                     + list(chain.from_iterable([("-r", r) for r in requirements]))
                     + manager_args
```

Ruff (0.15.20)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)